### PR TITLE
Update add/remove classes to accept array of elements and classes

### PR DESCRIPTION
**Pull request by mturnwall**

### CODE REVIEW

File: lib/addClass.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+import find from './find';
+
+function addClass(selector, cls) {
+  if (Array.isArray(selector)) {
+    return selector.forEach((item) => addClass(item, cls));
+  }
+  const els = find(selector);
+  if (els.length) {
+    els.forEach((el) => {
+      if (Array.isArray(cls)) {
```
Comment (jgallen23) [inline lib/addClass.js]: just convert cls to always be a class if its not, saves you a check

File: lib/removeClass.js [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
+function removeClass(selector, cls) {
+  if (Array.isArray(selector)) {
+    return selector.forEach((item) => removeClass(item, cls));
+  }
+
+  const els = find(selector);
+  if (els.length) {
+    els.forEach((el) => {
+      if (Array.isArray(cls)) {
```
Comment (jgallen23) [inline lib/removeClass.js]: same here

**Review by jgallen23 — CHANGES_REQUESTED** [review]

**Comment by mturnwall** [comment]
I wasn't exactly sure what you meant by converting `cls` to always be a class. So just always convert `cls` to be an array instead.

`const clsArray = [].concat(cls);`

**Comment by jgallen23** [comment]
You should do the conversion higher up. Inefficient right now to do it inside the loop

**Comment by jgallen23** [comment]
And need merge

**Pull request merged**